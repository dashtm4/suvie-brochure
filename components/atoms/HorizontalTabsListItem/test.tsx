import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { HorizontalTabsListItem } from '.'

describe('<HorizontalTabsListItem />', () => {
  const children = 'Test children'

  it('should renders correct <HorizontalTabsListItem />', () => {
    const wrapper = shallow(
      <HorizontalTabsListItem>{children}</HorizontalTabsListItem>
    )

    expect(wrapper.contains(children)).toBeTruthy()
    expect(wrapper.prop('className')).toContain('text-white')

    expect(toJSON(wrapper)).toMatchSnapshot()
  })

  it('should has correct className with light theme', () => {
    const theme = 'light'

    const wrapper = shallow(
      <HorizontalTabsListItem theme={theme}>{children}</HorizontalTabsListItem>
    )

    expect(wrapper.prop('className')).toContain('text-black')

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
