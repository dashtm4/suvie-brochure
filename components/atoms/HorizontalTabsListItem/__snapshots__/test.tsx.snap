// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HorizontalTabsListItem /> should has correct className with light theme 1`] = `
<Tab
  className="text-center md:text-left text-opacity-25 pb-2 md:px-2 cursor-pointer flex-grow text-black"
  disabledClassName="react-tabs__tab--disabled"
  focus={false}
  id={null}
  panelId={null}
  selected={false}
  selectedClassName="react-tabs__tab--selected"
  tabIndex="0"
>
  Test children
</Tab>
`;

exports[`<HorizontalTabsListItem /> should renders correct <HorizontalTabsListItem /> 1`] = `
<Tab
  className="text-center md:text-left text-opacity-25 pb-2 md:px-2 cursor-pointer flex-grow text-white"
  disabledClassName="react-tabs__tab--disabled"
  focus={false}
  id={null}
  panelId={null}
  selected={false}
  selectedClassName="react-tabs__tab--selected"
  tabIndex="0"
>
  Test children
</Tab>
`;
